ios/RNCWebView.m

```diff
@@ -251,6 +251,9 @@ - (void)didMoveToWindow
     _webView.UIDelegate = self;
     _webView.navigationDelegate = self;
 #if !TARGET_OS_OSX
+    if (_pullToRefreshEnabled) {
+        [self addPullToRefreshControl];
+    }
     _webView.scrollView.scrollEnabled = _scrollEnabled;
     _webView.scrollView.pagingEnabled = _pagingEnabled;
     _webView.scrollView.bounces = _bounces;
```

Comment:
For `UIRefreshControl` to work correctly, the `bounces` should always be `true`

Reply:
> For `UIRefreshControl` to work correctly, the `bounces` should always be `true`

Ok, will do today a little bit later

Reply:
> For `UIRefreshControl` to work correctly, the `bounces` should always be `true`

Done. I'm keeping the actual value of `bounces` in `_bounces` so that it could be changed if user wants to remove PTR, also added references to this behaviour to avoid possible confusion.